README.md

```diff
@@ -24,15 +24,19 @@ You will have to set up your credentials as described [here](https://cloud.googl
 
 ## Download
 
-Then, you can run the download script:
+Then, you can run the download script
```

Comment:
The colon is more standard. Up to the point that restructured text literally is harder to write without it.

Reply:
OK, reverting back.